Multiple idps (#205)
mulesoft/docs-api-manager: modules/ROOT/pages/create-instance-task.adoc

> = Creating an API instance

Reviewer: ```suggestion
= Creating an API Instance
```

Reviewer: Because this topic actually contains the steps, I think it should be titled "Create an API Instance."

Author: Done! Thanks!

ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

> In API Manager 2.x, you can use instances of an API in multiple environments. Additionally, you can also have multiple instances in the same environment. 

Reviewer: Redundant; omit either "Additionally" or "also." You do not need both.

Author: Done! Thanks!

> You can create one instance of an API that serves as a proxy. You can create another to manage directly as a basic endpoint. You can apply caching policies, for example to the proxy, and throttling policies and security policies to the basic endpoint.

Reviewer: ```suggestion
You can create one instance of an API that serves as a proxy. You can create another to manage directly as a basic endpoint. You can apply caching policies (for example, to the proxy), and throttling and security policies to the basic endpoint.
```

Author: Done! Thanks!

After setting up an API version for management, you repeat the process to create another instance. Attempting to set up the same version creates another instance of the version. You can create multiple instances and expand the version information to view details about them.

To create a API instance:

. Select *API Administration > Manage API* to create an API instance from Exchange.
+

> If you attempt to create two API verions of the same name, an instance of the existing API is automatically created. For example, set up version 1.0 of an API named jsonplaceholder.:

Reviewer: ```suggestion
If you attempt to create two API versions with the same name, an instance of the existing API is automatically created. For example, to set up version 1.0 of an API named `jsonplaceholder`:
```

Reviewer: I think that this text needs clarification: are you setting up two new APIs with the same name, or are you setting up one new version as a copy of the existing version, both with the same name?

Reviewer 2: Agree that I don't completely understand.

Author: Removed the sentence!

+
image::manage-exchange-end-task.png[]
+

> . From *API Name*, select the same API from Exchange, for example, jsonplaceholder.

Reviewer: ```suggestion
. From *API Name*, select the API (`jsonplaceholder`).
```

Author: Done! Thanks!

> . From *API Version*, select the same version of the API, for example 1.0.

Reviewer: ```suggestion
. From *API Version*, select version (`1.0`).
```

Author: Done! Thanks!

. From *Asset Version*, select the Exchange asset version.

> . From *Managing Type*, choose Basic Endpoint or Endpoint with Proxy. 

Reviewer: ```suggestion
. From *Managing Type*, select *Basic Endpoint* or *Endpoint with Proxy*. 
```

Author: Done! Thanks!

+

> For example, select Endpoint With Proxy.

Reviewer: ```suggestion
For this example, select *Endpoint With Proxy*.
```

Author: Done! Thanks!

> . In *Implementation URI*, enter the URL that hosts your web site or an arbitrary URL, such as the example shown `+http://api.domain.com+`. 

Reviewer: ```suggestion
. In *Implementation URI*, enter the URL that hosts your web site or an arbitrary URL, such as the example shown (`+http://api.domain.com+`). 
```

Author: Done! Thanks!

+

> An entry, any entry, is required if you chose Endpoint with Proxy. You cannot accept the default example; otherwise, saving this configuration is disabled.

Reviewer: If you choose *Endpoint with Proxy* and do not make an entry (such as accepting the default value), you cannot save your configuration.

Author: Done! Thanks!

. From *Client provider*, select the IdP that you want to use to authenticate access to this API from registered client applications:
+
image::create-instance-endpoint-midp.png[95%,95%]

> +

Reviewer: BREAK HERE to prevent GitHub issues.

If no client providers are set up for your environment and organization, this field is not visible.

> . Select a proxy deployment target, for example CloudHub.

Reviewer: ```suggestion
. Select a proxy deployment target, for example, CloudHub.
```

Author: Done! Thanks!

> . Accept the default Path `/` for the proxy endpoint or enter another path.

Reviewer: ```suggestion
. Accept the default path `/` for the proxy endpoint, or enter another path.
```

Author: Done! Thanks!
